README.md

```diff
@@ -1,7 +1,6 @@
 # node-coveralls
 
 [![Build Status][travis-image]][travis-url] [![Coverage Status][coveralls-image]][coveralls-url]
-[![Known Vulnerabilities](https://snyk.io/test/github/nickmerwin/node-coveralls/badge.svg)](https://snyk.io/test/github/nickmerwin/node-coveralls)
```

Comment:
Not sure if there's another badge we could use here. If there is, do let me know and I'd be happy to add it.